game/fs.cc

```diff
@@ -77,6 +77,25 @@ namespace {
 		return ret;
 	}
 
+	nlohmann::json readJSON(fs::path const& filename) {
+		nlohmann::json json;
+		std::clog << "json/debug: Will try to parse JSON in: " << filename.string() << std::endl;
+		std::ifstream file;
+			try {
+			file.open(filename.string());
+			if (!file.is_open()) throw std::runtime_error("Can't open file.");
+				json = nlohmann::json::parse(file);
+			} catch(nlohmann::json::exception const& e) {
+				std::clog << "json/error: " << e.what() << std::endl;
+			} catch(std::exception const& e) {
+				std::clog << "fs/error: " << e.what() << std::endl;
+			}
+			if (file.is_open()) { 
```

Comment:
No need to check if it's open. Just always close it :)

Reply:
I thought it would cause an error if it wasn't open but you're correct.